dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/ApplicationConfig.java

```diff
@@ -150,10 +151,18 @@ public String getEnvironment() {
     }
 
     public void setEnvironment(String environment) {
-        checkName("environment", environment);
+        checkName(EnvironmentConstants.ENVIRONMENT, environment);
         if (environment != null) {
-            if (!("develop".equals(environment) || "test".equals(environment) || "product".equals(environment))) {
-                throw new IllegalStateException("Unsupported environment: " + environment + ", only support develop/test/product, default is product.");
+            if (!(EnvironmentConstants.DEVELOPMENT_ENVIRONMENT.equals(environment)
+                    || EnvironmentConstants.TEST_ENVIRONMENT.equals(environment)
+                    || EnvironmentConstants.PRODUCTION_ENVIRONMENT.equals(environment))) {
+
+                throw new IllegalStateException(String.format("Unsupported environment: %s, only support %s/%s/%s, default is %s.",
+                         environment

```

Comment:
Hi, this code style looks a bit strange, I suggest writing the comma after the variable, like this:

```java
throw new IllegalStateException(String.format("Unsupported environment: %s, only support %s/%s/%s, default is %s.",
        environment,
        EnvironmentConstants.DEVELOPMENT_ENVIRONMENT,
        EnvironmentConstants.TEST_ENVIRONMENT,
        EnvironmentConstants.PRODUCTION_ENVIRONMENT,
        EnvironmentConstants.PRODUCTION_ENVIRONMENT));
```

What do you think?

Reply:
sure.